l10n_th_tier_department/tests/test_tier_validation_department.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2023 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo_test_helper import FakeModelLoader
+
+from odoo.tests.common import TransactionCase, tagged
+
+
+@tagged("post_install", "-at_install")
+class TierTierValidationDepartment(TransactionCase):
```

Comment:
```suggestion
from odoo.addons.base_tier_validation.tests.common import CommonTierValidation

class TierTierValidationDepartment(CommonTierValidation):
```

I think we not need to use `odoo_test_helper` because `base_tier_validation` module has CommonTierValidation class to inherit